packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/DecisionTableHeadEditorSpec.js

```diff
@@ -14,64 +14,149 @@ import ModelingModule from 'src/features/modeling';
+        DecisionTableHeadEditorModule,
+        ModelingModule
+      ],
+      debounceInput: false
+    }));
 
-    // then
-    expect(nameEl).to.exist;
-    expect(nameEl.textContent).to.eql('reason');
-  });
 
+    it('should render output name', function() {
 
-  it('should render output label', function() {
+      // when
+      const nameEl = domQuery('.output-name', testContainer);
 
-    // when
-    const labelEl = domQuery('.output-label', testContainer);
+      // then
+      expect(nameEl).to.exist;
+      expect(nameEl.textContent).to.eql('reason');
+    });
+
+
+    it('should render output label', function() {
+
+      // when
+      const labelEl = domQuery('.output-label', testContainer);
+
+      // then
+      expect(labelEl).to.exist;
+      expect(labelEl.textContent).to.eql('Check Result');
+    });
 
-    // then
-    expect(labelEl).to.exist;
-    expect(labelEl.textContent).to.eql('Check Result');
-  });
 
+    it('should render input expression', function() {
 
-  it('should render input expression', function() {
+      // when
+      const expressionEl = domQuery('.input-expression', testContainer);
 
-    // when
-    const expressionEl = domQuery('.input-expression', testContainer);
+      // then
+      expect(expressionEl).to.exist;
+      expect(expressionEl.textContent).to.eql('sum');
+    });
 
-    // then
-    expect(expressionEl).to.exist;
-    expect(expressionEl.textContent).to.eql('sum');
+
+    it('should render input label', function() {
+
+      // when
+      const labelEl = domQuery('.input-label', testContainer);
+
+      // then
+      expect(labelEl).to.exist;
+      expect(labelEl.textContent).to.eql('Customer Status');
+    });
   });
 
 
-  it('should render input label', function() {
+  describe('updating values', function() {
+
+    beforeEach(bootstrapModeler(diagramXML, {
+      modules: [
+        CoreModule,
+        DecisionTableHeadModule,
+        DecisionTableHeadEditorModule,
+        ModelingModule,
+        {
+          debounceInput: [ 'factory', flushableDebounceInputFactory ]
+        }
+      ],
+    }));
+
+
+    it('should correctly display erased input expression', inject(
+      function(debounceInput) {
+
+        // given
+        const inputCell = domQuery('th[data-col-id="input2"]', testContainer);
+        inputCell.click();
 
-    // when
-    const labelEl = domQuery('.input-label', testContainer);
+        const input = domQuery('.ref-input-label', testContainer);
 
-    // then
-    expect(labelEl).to.exist;
-    expect(labelEl.textContent).to.eql('Customer Status');
+        setValue(input, 'to-erase');
+        debounceInput.flush();
+
+        // when
+        setValue(input, '');
+
+        // then
+        expect(input.value).to.eql('');
+      })
+    );
+
+
+    it('should correctly display erased output expression', inject(
+      function(debounceInput) {
+
+        // given
+        const inputCell = domQuery('th[data-col-id="input2"]', testContainer);
```

Comment:
It seems like there's a copy-paste mistake here :P Could you convert the inputCell to outputCell, update the selector? We're testing input expression two times. So undoing changes in OutputCellContextMenu.js does not affect the test results.

Reply:
Done.